core-services/prow/02_config/openshift/api/_pluginconfig.yaml

```
@@ -18,3 +18,22 @@ plugins:
   openshift/api:
     plugins:
     - approve
+    - welcome
+welcome:
+- always_post: true
+  message_template: |
+    Hello @{{.AuthorLogin}}! Some important instructions when contributing to {{.Org}}/{{.Repo}}:
+    In addition to the standard ``lgtm`` and ``approved`` labels this repository requires either:
+
+    ``bugzilla/valid-bug`` - applied if your PR references a valid bugzilla bug
+
+    OR
+
+    ``qe-approved``, ``docs-approved``, and ``px-approved`` - these labels can be applied by anyone in the openshift org via the ``/label`` command.
+
+    Who should apply these qe/docs/px labels?
+    - For a no-Feature-Freeze team who is merging a feature before code freeze, they need to get those labels applied to their api repo PR by the appropriate teams (i.e. qe, docs, px)
+    - For a Feature Freeze(traditional) team who is merging a feature before FF, they can self-apply the labels(via /label commands), they are basically irrelevant for those teams
```

Nit, should have a couple of extra spaces
```suggestion
    - For a Feature Freeze (traditional) team who is merging a feature before FF, they can self-apply the labels (via /label commands), they are basically irrelevant for those teams
```

---

fixed